guidelines/curation_guidelines.md

```diff
@@ -0,0 +1,30 @@
+# Guidelines for CELLxGENE Curators
+
+Identifying Data to Curate
+----------------
+Once a curator identifies a study for CELLxGENE submission, they should also identify any secondary DOIs, the publication DOIs corresponding to any data that the primary study includes. The curator should then check the [Dataset sheet](https://docs.google.com/spreadsheets/d/1ax9b5sxmxSJgrjncXG5WGilgIGKm2EEWmILpoN6pLzY/edit?usp=sharing) to see if the primary study has already been submitted or is currently being curated. The curator should also check the [reused data tab](https://docs.google.com/spreadsheets/d/1ax9b5sxmxSJgrjncXG5WGilgIGKm2EEWmILpoN6pLzY/edit?usp=sharing) for the primary study and both tabs for each secondary DOI to see if any data from the study have been included in other CELLxGENE submissions. Data that are already included can likely still be submitted, but the data redundancy should be discussed with the Lattice team as it will have implications on how to curate [`is_primary_data`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#is_primary_data). The curator should email the [Lattice team](mailto:lattice-info@lists.stanford.edu) with the DOI, as well as the secondary DOIs, so that the sheets can be updated and any data redundancies can be addressed.
+
```

Comment:
RE _currently being curated_ - I would not expect the Datasheet sheet to expose private collections so the comment is confusing. I could imagine that the sheet might include the DOI and curator name for in progress items.